requirements.txt

```
@@ -8,7 +8,7 @@ mccabe==0.6.1
 pathspec==0.8.0
 pycodestyle==2.6.0
 pyflakes==2.2.0
-PyYAML==5.3.1
+PyYAML==5.4
```

```suggestion
PyYAML==5.4.1
```
Might as well bump to patch